src/ansys/fluent/core/logging.py

```diff
@@ -170,9 +170,7 @@ def configure_env_var() -> None:
             env_logging_level = int(env_logging_level)
         else:
             env_logging_level = env_logging_level.upper()
-        if env_logging_level in [0, "OFF"] or is_active():
-            pass
-        else:
+        if not (env_logging_level in [0, "OFF"] or is_active()):
```

Comment:
simpler

Reply:
``` python
if not is_active() and env_logging_level not in [0, "OFF"]:
  print(...)
```

Reply:
In my understanding all three ways are equivalent, but the one by @h-krishnan is easier to understand. 

Adding a commit suggestion for @h-krishnan's idea to make it easier to incorporate, if you prefer:
```suggestion
        if not is_active() and env_logging_level not in [0, "OFF"]:
```

Reply:
> ```python
> if not is_active() and env_logging_level not in [0, "OFF"]:
>   print(...)
> ```

Yes, I would have done the same - minimising keystrokes